plugins/woocommerce/readme.txt

```diff
@@ -160,6 +160,6 @@ WooCommerce comes with some sample data you can use to see how products look; im
 
 == Changelog ==
 
-= 6.8.0 2022-XX-XX =
```

Comment:
In https://github.com/woocommerce/woocommerce/commit/a48f3ef34c1a42d6a9664a4dfb1ce1cfb22c0e24 there is also a bump of the "stable" tag on line 7. Do you want to change that as well?